File: test/fixtures/python/multipart_post.py
import requests

headers = {
    'Authorization': 'Bearer ACCESS_TOKEN',
}

files = {
    'attributes': (None, '{"name":"tigers.jpeg", "parent":{"id":"11446498"}}'),
    'file': open('myfile.jpg', 'rb'),
}

response = requests.post('http://localhost:28139/api/2.0/files/content', headers=headers, files=files)

File: test/fixtures/julia/multipart_post.jl
using HTTP

headers = Dict(
    "Authorization" => "Bearer ACCESS_TOKEN"
)

form = HTTP.Form(
    Dict(
        "attributes" => "{\"name\":\"tigers.jpeg\", \"parent\":{\"id\":\"11446498\"}}",
        "file" => open("myfile.jpg")
    )
)

resp = HTTP.post("http://localhost:28139/api/2.0/files/content", headers, form)
